velox/functions/sparksql/aggregates/ModeAggregate.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <string>
+
+namespace facebook::velox::functions::aggregate::sparksql {
+
+void registerModeAggregate(
```

Comment:
This header only provides a function for the registration. Perhaps follow below to use extern instead and this header can then be removed.

https://github.com/facebookincubator/velox/blob/c8ac8f7abeaebcc0359e17fa1199b5f59f549a6d/velox/functions/sparksql/aggregates/Register.cpp#L29-L32